tests/conftest.py

```diff
@@ -149,3 +150,11 @@ def build_pkg(dist_dir: Path, of: Path, distributions: Sequence[DistributionType
 @pytest.fixture(scope="session")
 def pkg_builder() -> Callable[[Path, Path, Sequence[DistributionType], bool], Path]:
     return build_pkg
+
+
+@pytest.fixture(scope="session", autouse=True)
+def no_default_config_ini() -> Iterator[None]:
+    filename = str(uuid4())
```

Comment:
use pytest-mock over monkeypatch

Reply:
Will do. I'll update this ASAP.

Reply:
I think that I've addressed this.

I didn't include a changelog fragment because the contributing doc stated "a changelog entry is meant for end users and should only contain details relevant to an end user", and this only affects developers. If a changelog fragment is needed I can add that.